project/Build.scala

```diff
@@ -507,4 +512,16 @@ object Build extends sbt.Build {
         )
       )
   )
+
+  lazy val pipeline = Project(
+    id = "gearpump-experiments-pipeline",
+    base = file("experiments/pipeline"),
+    settings = commonSettings ++ myAssemblySettings ++
+      Seq(
+        libraryDependencies ++= Seq(
+          "org.apache.hbase" % "hbase-client" % clouderaHBaseVersion,
+          "org.apache.hbase" % "hbase-common" % clouderaHBaseVersion
```

Comment:
We may want to combine them. The pipeline processor has insert, findByKey, deleteByKey, insert, update. I could address as a separate issue to make the HBaseSink support a number of operations and configuration zookeeper, etc. We may also want to have it optionally create the table, columnFamily.